worrdle/src/App.js

```diff
@@ -1,20 +1,174 @@
+    if(shouldGameStop){
+      setGameState(false); // set game state to false and the game is over
+    }
+    else{
+      if(activeRow+1<6){
+        setActiveRow(activeRow+1); // increment the active row
+        setActiveCol(0); // set active col to 0
+      }else{
+        setGameState(false); // set game state to false and the game is over
+      }
+    }
+}
+
+// function that Deletes the word on calling delete
+const DeleteWord = () => {
+
+  // check if the current game is not over
+  if(!gameState) return;
+
+  // if current cell is not empty 
+  if(newboard[activeRow][activeCol]){
+    newboard[activeRow][activeCol]=''; // assign it the empty string
+    setNewBoard(newboard); // set the new state of the board
+    if(activeCol-1>=0){
+      setActiveCol(activeCol-1);
+    }
+  }
+
+  // if current cell is empty then delete it's previous cell.
+  else if(activeRow >=0  && activeRow < 6){
+      if (activeCol >=0 && activeCol < 5){
+          if(activeCol!==0){ // edge case check if its not the very first cell.
+              newboard[activeRow][activeCol-1]=''; // assign it the empty string
+              setNewBoard(newboard); // set the new state of the board
+              setActiveCol(activeCol - 1);
+          }else{
+              newboard[activeRow][activeCol]=''; // assign it the empty string
+              setNewBoard(newboard); // set the new state of the board
+          }
+      }
+  }
+};
+
+// keyHandler if for adding the functionality to the letter keyboard buttons
+const keyHandler = (props) => {
+
+  if(!gameState) return; // if the game is over then return;
+  
+  let updateBoard = []; // to store the updated board
+  
+  if(newboard[activeRow][activeCol]!==''){
+    setActiveCol(activeCol+1);
+  }
+
+  // make the updations in the newboard
+  for(let i = 0; i<newboard.length; ++i){
+      let arr =[];
+      for(let j = 0; j<newboard[i].length;++j){
+        // if the current cell was updated then update in the new board as well
+        if(i===activeRow && j===activeCol){
+          arr.push(props.Keys[props.id - 1][props.key-1]);
+        }else{
+          // else push the same state in the new board
+          arr.push(newboard[i][j]);
+        }
+      }
+      updateBoard.push(arr);
+  }
+  // set the state of the new board
+  setNewBoard(updateBoard);
+
+  // lets update the active row and col
+  let updatedActiveCol = activeCol;
+  let updatedActiveRow = activeRow;
+
+  // if the active column can be incremented, increment it.
+  if(activeCol+1<5){
+      updatedActiveCol = activeCol+1;
+  }
+
+  // set the active Row and Col
+  setActiveRow(updatedActiveRow);
+  setActiveCol(updatedActiveCol);
+};
+
 
   return (
     <>
-    <div>
-      <h1 style = {{color: 'white', justifyContent: 'center', alignItems:'center', marginLeft: 390}}> Worrdle</h1>
+    <div >
+      <h1 className='Title' style = {{color: 'white', textAlign: 'center', fontWeight: 'bold', fontFamily: 'Arial'}}> Worrdle</h1>
```

Comment:
It is not a really good idea to create anonymous style objects because of performance.
Instead assign classnames and use those classnames to achieve stylings.